src/Schema/Comparator.php

```diff
@@ -71,7 +77,7 @@ public function compare(Schema $fromSchema, Schema $toSchema)
             if (! $fromSchema->hasTable($tableName)) {
                 $diff->newTables[$tableName] = $toSchema->getTable($tableName);
             } else {
-                $tableDifferences = $this->diffTable(
+                $tableDifferences = $this->compareTables(
```

Comment:
This has to remain unchanged. Somebody may use a customized version of the comparator with the overridden `diffTable()` implementation. If the library code stops calling their customized code, it may be considered a BC break. The library code is allowed to use its deprecated APIs.